### ISSUE TITLE
camera preview not visible when part of Swiper

### ISSUE THREAD

**Issue by maddanio** [react-native-community/react-native-camera#585]
### Steps to reproduce
1.

 put camera on a swiper card from react-native-swiper along at least one other:

```
<Swiper>
     <Camera ...>
     <View...>
</Swiper>

```

2.

run the app

### Expected behaviour
The preview should show what the camera sees

### Actual behaviour
The preview shows up with a seemingly random color, i.e. black or white in my tests, probably from some other style

### Environment
- **Node.js version**: v7.5.0
- **React Native version**: 0.37.0
- **React Native platform + platform version**: iOS 10.2

### react-native-camera
**Version**: 1.0.0-alpha, but also with older versions

**Comment by maddanio** [comment]
since react-native-swiper seems to be very popular I guess this could be a big issue for anyone wanting to "swipe to camera" like instagram etc.

**Comment by maddanio** [comment]
ok, digging now. i just found out that when put into the swiper the camera view seems to be created twice (for whaterver reason). I could imagine that being a problem, since the view is actually created only once, as is the preview layer.

**Comment by maddanio** [comment]
interestingly that does not happen to the mapbox view...

**Comment by maddanio** [comment]
btw, i am currently blabbering about this both here and on react-native: [https://github.com/facebook/react-native/issues/12262](url)

**Comment by serranoarevalo** [comment]
Hey, happened to me too when I was trying to make a snapchat clone. What worked for me was having `loadMinimal={true}` as a prop in the Swiper. 

For example:

`<Swiper ref='swiper' horizontal={false} loop={false} showsPagination={false} index={0} loadMinimal={true}></Swiper>`

**Comment by maddanio** [comment]
ok, that works too it seems, thanks!
still leaving this open as it should work anyhow. My plan of triaging it was to make a completely new native view and bit by bit copy code in from the camera view. but now i will postpone that I guess :)

**Comment by ptomasroos** [comment]
Why would you want to leave this as open @maddanio ? This feels like it belongs as a problem of the Swiper implementation?

**Comment by maddanio** [comment]
Not convinced, since other views have no such problems, including mapbox

**Comment by SimonErm** [comment]
This issue is closed because of inactivity.
Furthermore RCTCamera is deprecated now, use RNCamera instead. If this issue is also related to RNCamera, please open a new one.

**Comment by asgerhallas** [comment]
@maddanio could seem like `<Swiper loadMinimal>` does the trick.
